### DISCUSSION TITLE
각자가 담당할 components 논의

### DISCUSSION THREAD

**Discussion by brightchul** [es6frontendstudy2021/cheese-ui-component#2] [General]
input, button, modal이 있을 텐데 각자 어떤 컴포넌트를 만들지 정해봅시다

@cheese-cake-cookie

**Comment by cheese-cake-cookie** [comment]
저는 button, modal은 하나씩 담당 input 태그는 형태가 다른 것들을 나누어 작업 한 후 한 디렉토리에 합치기 방식을 생각했는데 어떠세요?

**Reply by brightchul** [reply]
네 그렇게 하면 되겠네요. 
input은 button, modal 부터 작업한 다음에 먼저 끝난 사람이 input type중 원하는 타입을 먼저 가져가는 식으로 하면 좋을거 같습니다.

**Comment by brightchul** [comment]
button, modal 중에서는 어떤 것을 해보고 싶으신가요?

**Reply by cheese-cake-cookie** [reply]
조금 고민해야 하는 부분이 많을 거 같긴 하지만 modal 한번 해보고 싶습니다

**Reply by brightchul** [reply]
네 그럼 저는 button을 하겠습니다.
